### ISSUE TITLE
Virtualizer: Export Virtualizer from react-aria

### ISSUE THREAD

**Issue by AlexVipond** [adobe/react-spectrum#10000]
### Provide a general summary of the feature here

**Background**: `Virtualizer` is written inside `react-aria`, but currently only exported from `react-aria-components`.

**Problem**: Some projects just install `react-aria`, not `react-aria-components`, usually due to a technical preference for React Aria's hooks. These projects cannot import `Virtualizer` unless they separately install `react-aria-components`.

**Solution**: Export `Virtualizer` from `react-aria`

### 🤔 Expected Behavior?

It should be possible to import `Virtualizer` from `react-aria`

### 😯 Current Behavior

It is not possible to import `Virtualizer` from `react-aria`

### 💁 Possible Solution

Export `Virtualizer` from `react-aria`

### 🔦 Context

See general summary.

The only thing I might add is a couple thoughts on the obvious question: why not just install `react-aria-components`, and let tree-shaking handle the rest?

Reasons:
- It's best practice for disk space and project cleanliness not to install packages unless they're absolutely necessary
- In some professional projects, there are strict security rules around installing new packages, and the process to approve a new package can be long

**Comment by devongovett** [comment]
FYI the Virtualizer in react-aria is different from the one in react-aria-components. The react-aria one is legacy. But will discuss this with the team.
